<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rahmen</class>
 <widget class="QDialog" name="Rahmen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rahmen</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>721</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Verzeichnis wählen</string>
    </property>
    <widget class="QLabel" name="la_dir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>430</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_autom">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_relaod">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>♻</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="la_deckel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>600</width>
      <height>420</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>600</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Zugehörige Bilder</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Bild:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>letzter Rahmen:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_1stP">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>430</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_lstR">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sel_1st">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Rahmen:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_1stR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_go">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_quit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>670</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abbruch</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>241</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Koordinaten der Punkte</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>rot oben links</string>
     </property>
     <widget class="QSpinBox" name="sb_r1x">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_r1y">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>rot oben rechts</string>
     </property>
     <widget class="QSpinBox" name="sb_r2x">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_r2y">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>rot unten links</string>
     </property>
     <widget class="QSpinBox" name="sb_r3x">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_r3y">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>grün</string>
     </property>
     <widget class="QSpinBox" name="sb_gx">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_gy">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pb_anzeigen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Bild anzeigen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_setK">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Übernehmen</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Bild drehen (Uhrzeigersinn)</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>90 Grad</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>180 Grad</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>270 Grad</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>550</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Deckel verschieben</string>
    </property>
    <widget class="QPushButton" name="pb_hoch">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string> ^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_links">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_rechts">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_runter">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
